.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 node_modules
```

Comment:
Yeah the 926 files.  Just having node_modules in there should omit anything with that directory name.   Maybe have to try with a trailing `/`?